### PR TITLE
CONSOLE-3965: i18n upload/download routine task for version-4.16/ sprint-248

### PR DESCRIPTION
**Pull request by cyril-ui-developer**
Adding latest translations from Memsource project https://cloud.memsource.com/web/project2/show/ey37gmOfw78ld93mF186Q2

### CODE REVIEW

File: frontend/packages/dev-console/locales/ja/devconsole.json [hunk 1]
```diff
@@ -627,14 +627,14 @@
   "Select query": "クエリーの選択",
   "Hide PromQL": "PromQL の非表示",
   "Show PromQL": "PromQL の表示",
-  "Dashboard": "ダッシュボード",
+  "Dashboards": "Dashboard",
```
Comment (TheRealJon) [inline frontend/packages/dev-console/locales/ja/devconsole.json:630]: Was this intentional?

**Review by TheRealJon — COMMENTED** [review]
Looks like one Japanese translation for "Dashboard" was removed.

**Comment by cyril-ui-developer** [comment]
@TheRealJon Updated, PTAL

File: frontend/packages/dev-console/locales/ja/devconsole.json (hunk 1 above)
Comment (cyril-ui-developer) [reply frontend/packages/dev-console/locales/ja/devconsole.json]: @TheRealJon I don't think so. Seems like a mistake to me. I have updated with the correct translation. 
PTAL, thanks.

**Review by rhamilto — APPROVED** [review]
/lgtm

**Comment by cyril-ui-developer** [comment]
Adding these labels since the changes are translations in supported langs.
/label docs-approved
/label px-approved
/label qe-approved

**Pull request merged**